packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -37,6 +38,13 @@ import LinkControl from '../link-control';
 import { store as blockEditorStore } from '../../store';
 
 const noop = () => {};
+
+const BUTTON_VARIANTS = {
+	default: ToolbarButton,
+	toolbar: ToolbarButton,
```

Comment:
Leaving the final review to @mirka, but just a side note here.

I'd personally remove `default` and just make `buttonVariant` default to `toolbar`. No need to maintain two variants that do the same thing.